.github/workflows/build.yml

```diff
@@ -108,3 +129,10 @@ jobs:
 
       - name: Linux Script
         run: sudo ./.travis/${{ matrix.os_name }}_runner.sh script
+
+      - name: Publish Artifact
+        if: ${{ startsWith(github.ref, 'refs/heads/v') && endsWith(matrix.os_name, 'linux_openresty') }}
```

Comment:
Should be `matrix.os_name == 'linux_openresty'`?

Reply:
Fixed. Thanks!